subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -845,6 +848,14 @@ void ull_conn_iso_start(struct ll_conn *conn, uint16_t cis_handle,
 	 * validated handle.
 	 */
 	if (cig->state == CIG_STATE_ACTIVE) {
+#if !defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+		cis_lazy_get(cis);
```

Comment:
`_get` implies that this returns a value, but that does not seem to be the case.

Reply:
renamed